CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ CHANGELOG for Sulu
 * dev-develop
     * ENHANCEMENT #695 [ContentBundle]  Hide textblock sort option when there is only 1 textblock available
     * FEATURE     #634 [AdminBundle]    Created new configuration component, added new configuration for autocomplete
+    * BUGFIX      #681 [TAGBUNDLE]      Fixed filtering of tags in Tag list and Media edit Overlay
+    * BUGFIX      #681 [MediaBundle]    Fixed imagick detection
+    * BUGFIX      #681 [TagBundle]      Fixed filtering of tags in Tag list and Media edit Overlay
+    * FEATURE     #634 [All]            Created new configuration component, added new configuration for autocomplete
```

Comment:
We really try to avoid All now, I would say the configuration component is part of the AdminBundle